tests/core/escaping/test.sh

```diff
@@ -3,14 +3,13 @@
 
 rlJournalStart
     rlPhaseStartSetup
+        rlRun "HOW=${PROVISION_HOW:-local}"
```

Comment:
If I may, I'd recommend logging it as it is named, `PROVISION_HOW=` instead of `HOW=...`. The "how" can be applied to all steps, there might be tests out there parametrizing also `report`, for example, who knows what "how" might mean. On the other hand, `PROVISION_HOW` is absolutely clear. 10 extra characters won't ruin our budget, and we'll get a log that's easy to follow without hesitation.

Reply:
Ack, fixed in 4cbe6a77.